portfolio/src/main/webapp/comments.js

```diff
@@ -76,12 +76,17 @@ function postComment(commentSubmissionEvent) {
 }
 
 /** 
- * Tells the servlet to delete the comment marked by @param commentObj and refreshes the page 
+ * Tells the servlet to delete the comment marked by @param commentObj or all comments if deleteAll is specified 
+ * and refreshes the page 
  * @param {Object} commentObj 
+ * @
  */
-function deleteCommentFromDB(commentObj) {
+function deleteCommentFromDB(commentObj, deleteAll = false) {
```

Comment:
I might have missed something, it's not use currently right? I think I'd prefer 2 separate functions if that makes sense. Since for deleting all the comments we don't need `commentObj`.

Reply:
Yeah you're right. I'll add it as a separate function